app/src/main/java/com/OxGames/Pluvia/service/handler/PluviaHandler.kt

```diff
@@ -0,0 +1,39 @@
+package com.OxGames.Pluvia.service.handler
+
+import com.OxGames.Pluvia.service.callback.EmoticonListCallback
+import `in`.dragonbra.javasteam.base.ClientMsgProtobuf
+import `in`.dragonbra.javasteam.base.IPacketMsg
+import `in`.dragonbra.javasteam.enums.EMsg
+import `in`.dragonbra.javasteam.protobufs.steamclient.SteammessagesClientserverFriends.CMsgClientGetEmoticonList
+import `in`.dragonbra.javasteam.steam.handlers.ClientMsgHandler
+import `in`.dragonbra.javasteam.steam.steamclient.callbackmgr.CallbackMsg
+
+/**
+ * Custom handler to handle dispatching that JavaSteam does not support.
+ */
+class PluviaHandler : ClientMsgHandler() {
+
+    companion object {
+        fun getCallback(packetMsg: IPacketMsg): CallbackMsg? = when (packetMsg.msgType) {
+            EMsg.ClientEmoticonList -> EmoticonListCallback(packetMsg)
+            else -> null
+        }
+    }
+
+    /**
+     * Handles a client message. This should not be called directly.
+     * @param packetMsg The packet message that contains the data.
+     */
+    override fun handleMsg(packetMsg: IPacketMsg) {
+        val callback = getCallback(packetMsg) ?: return
+
+        client.postCallback(callback)
```

Comment:
Preventing potential crash when client is null (only relevant if this even can be null)

```suggestion
        getCallback(packetMsg)?.let { callback ->
            client?.postCallback(callback)
        }
```

Reply:
This the same style applied in JavaSteam's handlers. So this wont be changed  due to consistency with JavaSteam, and the syle of SteamKit